### PR TITLE
RAINCATCH-1259 Signature Common Step 

### PR DESCRIPTION
**Pull request by austincunningham**
## Motivation:
Need to documentation of to the Signature Common Step and common steps in general

## What:
Description of common step and its usage. 

- [ ] Documentation

### CODE REVIEW

File: docs/workforce-management-framework/topics/con-wfm-common-steps.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+= WFM Common Steps
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/con-wfm-common-steps.adoc]: @jstaffor Leaving you with options how we going to call them. 
Community suggestions were:
- Out of the box
- Generic
- Common
- Build in

**Review by jstaffor — CHANGES_REQUESTED** [review]
I reduce the content quite a bit. Defo reread it. Also, Basic Step seemed like the best name for me - wdyt? If you don't like it, that's fine with me.

**Comment by wtrocki** [comment]
@jstaffor - basic weren't in my options and it's not carry any meaning. - Basic = Simple. 
We looking for `Reusable`, `Generic` etc.

**Review by jstaffor — APPROVED** [review]
Change made - it looks good to me guys

**Comment by wtrocki** [comment]
Great. @austincunningham feel free to merge once youi are ok as well.

**Comment by wtrocki** [comment]
@jstaffor 
-1 For basic step.

Final agreement was to go with: Packaged Step

**Pull request merged**